src/components/NavigationBar/NavigationBar.stories.js

```diff
@@ -0,0 +1,65 @@
+import NavigationBar from '.';
+import { withRouter } from 'storybook-addon-react-router-v6';
+
+export default {
+	title: 'Components/NavigationBar',
+	component: NavigationBar,
+	decorators: [withRouter],
+	parameters: {
+		tags: ['autodocs'],
+	},
+	argTypes: {
+		mode: {
+			type: 'string',
+			description: 'Вид меню навигации',
+			defaultValue: 'header',
+			options: ['header', 'footer'],
+			control: {
+				type: 'radio',
+			},
+		},
+
+		variant: {
+			type: 'string',
+			description: 'Параметры шрифта',
+			options: ['body1', 'body2'],
+			control: {
+				type: 'radio',
+			},
+		},
+
+		color: {
+			type: 'string',
+			description: 'Цвет текста',
+			options: ['black', 'grey5'],
+			control: {
+				type: 'radio',
+			},
+		},
+	},
+};
+
+const Template = (args) => <NavigationBar {...args} />;
+
+const arrLinks = [
```

Comment:
Обычно для констант используется uppercase, если они «жёстко закодированы»
```suggestion
const ARR_LINKS = [
```

Reply:
тут еще